private/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package private contains helpers private to github.com/containers/image, with no intent or commitment to make them useful for any other purpose.
+// Do not use outside!
+package private
```

Comment:
w/o `private` component and probably under `transport` (IIUC they're related to transports, `explicitfilepath`) and/or `docker` (`dockerpolicyconfiguration` calling it just `policyconfiguration`)